### ISSUE TITLE
QoL request: Connect to view logs automatically/remember selected connection method

### ISSUE THREAD

**Issue by adamzimnyy** [esphome/dashboard#53]
After a recent update a dialog with more options to connect to devices appeared in the dashboard. 
With this change, connecting to your device to view logs requires choosing the method each time, even when the only option is to connect wirelessly. 

I would like the logs to open right away without needing to click an additional time on this dialog: 

![image](https://user-images.githubusercontent.com/14812161/127323827-6c644e4e-101e-4f36-8ebf-63a539fe312e.png)

If there is only one option available this is just an unneccesary step.

**Comment by balloob** [comment]
I agree. I think that in the future we can even go another step further and add this choice in the header of the dialog.
